protos/guard/guard.proto

```diff
@@ -141,6 +154,15 @@ message FileStoreStatus {
     (gogoproto.stdtime) = true
   ]; //the time for generate the status
   bytes guard_signature = 8;
+  enum RentalState {
+    NEW = 0; //the rental was first initialized
+    PARTIAL_NEW = 1; //part of the file status was new, indicate that some contracts were rebuild
+    EXPIRED = 2; //the file rental was expired, indicate that all shards can be removed from host
+    RENEW = 3; //the contract got renewed
+  }
+  RentalState rental_state = 9;
+  string preparer_pid = 10; //the stakeholder who prepared such file, if the file is NEW state, the preparer is rental
+  bytes preparer_signature = 11; //preparer's signature for FileStoreMeta
```

Comment:
Just a question, is the preparer_signature for signing the FileStoreMeta?

Reply:
yes, it used while guard complete the repair, guard need to sign the contract to represent the renter. And it can be used in the contract renew scenario.